src/main/java/ChatBot.java

```diff
@@ -0,0 +1,70 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+/**
+ * Represents the ChatBot object that the user interacts with.
+ * corresponds to UI, Tasklist, Storage, Parser classes that aids in its functions.
+ */
+public class ChatBot {
+    // class for the chat-bot for the Duke Project
+
+
+    private UI ui = new UI();
+    private TaskList taskLists;
+    private String fileLocation = "./Data/Tasks.txt"; //hard-coded relative file location of stored tasks
+    private Storage storage = new Storage();
+    private Parser parser = new Parser();
+
+
+    public ChatBot() {
+
+    }
+
+    // constructor for chat-bot to initialise the file that was saved, if does not exist, then create new one
+    public ChatBot(TaskList list) {
+        this.taskLists = list;
+    }
+
+    /**
+     * Function to run chat-bot and terminate when needed.
```

Comment:
In method header comments, the first sentence should start in the form Returns ..., Sends ..., Adds ... (not Return or Returnning etc.). So instead of starting with the word "Function", you can start with "Runs chat-bot...". There should also be an empty line between description and parameter section.